src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -92,7 +93,7 @@ export class NetCoreDebugHelper implements DebugHelper {
 
         const additionalProbingPathsArgs = NetCoreDebugHelper.getAdditionalProbingPathsArgs(platformOS);
 
-        const containerAppOutput = netSdkDebugHelper.isDotNetSdkBuild(debugConfiguration?.preLaunchTask)
+        const containerAppOutput = netSdkDebugHelper.isDotNetSdkBuild(debugConfiguration)
             ? appOutput
             : NetCoreDebugHelper.getContainerAppOutput(debugConfiguration, appOutput, platformOS);

```

Comment:
Can you implement an overridable method in NetCoreDebugHelper that can be used to implement different behavior between standard NetCore and the SDK? That would remove the need to explicitly check for SDK build or not here as the appropriate helper logic could be applied.